index.js

```diff
@@ -11,6 +11,26 @@ function format(obj) {
     : JSON.stringify(obj);
 }
 
+function isArrayOfTables(simplePairs) {
+  var isArrayOfTables = false;
+  forEach(simplePairs, function(array) {
+    var value = array[1];
+    if ((Array.isArray(value) && isPlainObject(value[0]))) {
+      isArrayOfTables = true;
+    }
+  });
+  return isArrayOfTables;
```

Comment:
This function can be simplified:

```js
function isArrayOfTables(simplePairs) {
  return simplePairs.some(function(array) {
    var value = array[1];
    return Array.isArray(value) && isPlainObject(value[0]);
  });
}
```

Reply:
function isArrayOfTables(simplePairs) {
  return simplePairs.some(function(array) {
    return Array.isArray(array[1]) && isPlainObject(array[1][0]);
  });
}

Reply:
Hi Kenan,

Is that ok?

Martin

Reply:
Hi Kenan, 

I updated the code based on your comments. Please review.

Thanks !